updater/spec/dependabot/updater_spec.rb

```diff
@@ -1292,7 +1265,8 @@ def expect_update_checker_with_ignored_versions(versions)
               service = build_service
               updater = build_updater(service: service, job: job)
 
-              allow(updater).to receive(:dependency_files).
+              # TODO: Move this stub unto Dependabot::DependencySnapshot once it is better integrated
+              allow(Dependabot::FileParsers).to receive_message_chain(:for_package_manager, :new, :parse).
```

Comment:
👍 

It's gonna be a chore to tackle all these TODOs, but I'm glad you're leaving them so we can take care of them.